paddle/fluid/memory/allocation/allocator_facade.cc

```diff
@@ -79,10 +80,11 @@ class AllocatorFacadePrivate {
         InitNaiveBestFitCUDAPinnedAllocator();
 #endif
 #ifdef PADDLE_WITH_ASCEND_CL
-        VLOG(3) << "npu num: " <<platform::GetNPUDeviceCount();
+        VLOG(3) << "npu num: " << platform::GetNPUDeviceCount();
         for (int dev_id = 0; dev_id < platform::GetNPUDeviceCount(); ++dev_id) {
           InitNaiveBestFitNPUAllocator(platform::NPUPlace(dev_id));
         }
+        InitNaiveBestFitNPUPinnedAllocator();
```

Comment:
As discussed offline, we can use NPUPinnedAllocator directly for the special purpose only